module/display_module.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pygame
+
+import main
```

Comment:
Files inside Modules usually should not call main.py!

This can cause Spaghetti code and it will be difficult to debug (also dangerous)! Either keep only the functions and move the important stuffs to Main.Py or create specific functions in Main.py to handle this file.

Reply:
Okay, Senior. I wanted to access a few global variables in the main.py file. Now I have transferred those variables to a global_config.py file and imported that file wherever needed. Fixed this issue.